k8s/overlays/dev/kustomization.yaml

```diff
@@ -6,9 +6,9 @@ resources:
 
 images:
   - name: musiek/runner
-    newTag: '0.16'
+    newTag: '0.17'
   - name: musiek/mediator
-    newTag: '0.2'
+    newTag: '0.21'
```

Comment:
Why 0.21 rather than 0.3?

Reply:
Just like in runner, I think with 0.1 incrementation we would reach 1.0 too early.